docs/3_guides/7_contract_addresses.md

```diff
@@ -1,6 +1,36 @@
 # Aligned contract addresses
 
-## Holesky deployments
+## Mainnet Deployments
+
+| Contract                   | Address                                                                                                               |
+|----------------------------|-----------------------------------------------------------------------------------------------------------------------|
+| AlignedLayerServiceManager | [0xeF2A435e5EE44B2041100EF8cbC8ae035166606c](https://etherscan.io/address/0xeF2A435e5EE44B2041100EF8cbC8ae035166606c) |
+| BlsApkRegistry             | [0x3CcfB7e6e8fe2A8d941a8Ce4C69A944a770E8228](https://etherscan.io/address/0x3CcfB7e6e8fe2A8d941a8Ce4C69A944a770E8228) |
+| IndexRegistry              | [0x9Bf1275e18eC8FA3cA7f9bffF1b0DF3e14C6E134](https://etherscan.io/address/0x9Bf1275e18eC8FA3cA7f9bffF1b0DF3e14C6E134) |
+| OperatorStateRetriever     | [0x6e0046205cAfA503F6b7465195A6C63C47d214f1](https://etherscan.io/address/0x6e0046205cAfA503F6b7465195A6C63C47d214f1) |
+| RegistryCoordinator        | [0xA8CC0749b4409c3c47012323E625aEcBA92f64b9](https://etherscan.io/address/0xA8CC0749b4409c3c47012323E625aEcBA92f64b9) |
+| StakeRegistry              | [0x45F5290a3630Cd6dc277B6f92227526121ca7c22](https://etherscan.io/address/0x45F5290a3630Cd6dc277B6f92227526121ca7c22) |
+| BatcherPaymentService      | [0xb0567184A52cB40956df6333510d6eF35B89C8de](https://etherscan.io/address/0xb0567184A52cB40956df6333510d6eF35B89C8de) |
+
+### Strategies
```

Comment:
I would add the official docs to the eigen strategies as well: https://github.com/Layr-Labs/eigenlayer-contracts/tree/testnet-holesky?tab=readme-ov-file#deployments